packages/jellyfish-api-core/__tests__/category/masternode/listMasternodes.test.ts

```diff
@@ -0,0 +1,51 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { MasternodePagination } from '../../../src/category/masternode'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should list masternodes', async () => {
+    const masternodes = await client.masternode.listMasternodes()
+
+    for (const masternode in masternodes) {
+      const createdMasternode = masternodes[masternode]
+      expect(typeof createdMasternode.ownerAuthAddress).toStrictEqual('string')
+      expect(typeof createdMasternode.operatorAuthAddress).toStrictEqual('string')
+      expect(typeof createdMasternode.creationHeight).toStrictEqual('number')
+      expect(typeof createdMasternode.resignHeight).toStrictEqual('number')
+      expect(typeof createdMasternode.resignTx).toStrictEqual('string')
+      expect(typeof createdMasternode.banHeight).toStrictEqual('number')
+      expect(typeof createdMasternode.banTx).toStrictEqual('string')
```

Comment:
I think you miss these 3 variables.

 ownerIsMine: false,
 operatorIsMine: false,
 localMasternode: false,

Reply:
Actually  i do not think `Masternodeinfo` contains those variables. Look at the screenshot with the `masternodeinfo` results

<img width="552" alt="image" src="https://user-images.githubusercontent.com/57817584/121661960-afea2a00-ca9c-11eb-9402-098cd4baa9d2.png">

Reply:
Hmm I got this

<img width="866" alt="Screenshot 2021-06-11 at 5 58 16 PM" src="https://user-images.githubusercontent.com/7101806/121669246-b4cfcd80-cade-11eb-99db-86a6b3e0aedf.png">

Reply:
Thanks explains why tests are failing in CI. Do you know why that is happening? Regardless, i will update it and check if it will pass in CI

Reply:
Okay, now tests are passing. Implemented